framework/midi/internal/zerberussynth.cpp

```diff
@@ -242,5 +247,20 @@ void ZerberusSynth::writeBuf(float* stream, unsigned int samples)
         return;
     }
 
-    m_zerb->process(samples, stream, nullptr, nullptr);
+    if (!isActive()) {
+        return;
+    }
+
+    if (m_preallocated.size() < samples) {
+        m_preallocated.resize(samples, 0);
+    }
+
+    std::fill(m_preallocated.begin(), m_preallocated.end(), 0.f);
+    m_zerb->process(samples, m_preallocated.data(), nullptr, nullptr);
+
+    for (unsigned int s = 0; s < samples; ++s) {
+        for (unsigned int c = 0; c < AUDIO_CHANNELS; ++c) {
+            stream[s * AUDIO_CHANNELS + c] = m_preallocated[s * (1 + c)];
+        }
+    }
```

Comment:
You can change zerberus process function,  Instead of doing double copying.
Here still 121212 https://github.com/musescore/MuseScore/blob/master/audio/midi/zerberus/voice.cpp#L297

Reply:
fixed